src/sensorthings.js

```diff
@@ -1,8 +1,24 @@
 import express from 'express';
 import base    from './routes/base';
+import things    from './routes/things';
+import locations    from './routes/locations';
+import historicalLocations    from './routes/historicalLocations';
+import featuresOfInterest    from './routes/featuresOfInterest';
+import datastreams    from './routes/datastreams';
+import sensors    from './routes/sensors';
+import observations    from './routes/observations';
+import observedProperties    from './routes/observedProperties';
 
 let router = express.Router();
 
 router.get('/', base);
+router.get('/Things', things);
+router.get('/Locations', locations);
+router.get('/HistoricalLocations', historicalLocations);
+router.get('/FeaturesOfInterest', featuresOfInterest);
```

Comment:
It's funny that all the endpoints are in plural except this one... It seems like a spec issue